tsfresh/feature_selection/selection.py

```diff
@@ -124,7 +128,20 @@ def select_features(X, y, test_for_binary_target_binary_feature=defaults.TEST_FO
 
         y = pd.Series(y, index=X.index)
 
-    df_bh = check_fs_sig_bh(X, y, n_jobs, chunksize, fdr_level, hypotheses_independent,
-                            test_for_binary_target_real_feature)
-
-    return X.loc[:, df_bh[df_bh.rejected].Feature]
+    if multiclass is True:
+        unique_labels = y.unique()
+        if len(unique_labels) < 3:
+            raise RuntimeError("Cannot perform multiclass selection on binary target")
```

Comment:
really? why? 

It can happen in a CV for a non binary target that the fold only contains two classes, this will break the whole cross validation. can we ditch that check?